app/controllers/application_controller.rb

```diff
@@ -1,7 +1,17 @@
 class ApplicationController < ActionController::Base
+  include Pundit
   before_action :authenticate_user!
-
   before_action :configure_permitted_parameters, if: :devise_controller?
```

Comment:
We want to add an after action here to make sure that we did call authorize and policy_scope after (almost) all actions
```rb
after_action :verify_authorized, unless: :devise_controller?
after_action :verify_policy_scoped, only: :index, unless: :devise_controller?
```

Reply:
Done!